src/app/homepage/pages/techniques/angular/angular.component.ts

```diff
@@ -0,0 +1,37 @@
+import { Component, ChangeDetectionStrategy } from '@angular/core';
+import { BasePageComponent } from '../../page/page.component';
+
+@Component({
+  selector: 'app-angular',
+  templateUrl: './angular.component.html',
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class AngularComponent extends BasePageComponent {
+
+  get notFoundExceptionFilter() {
+    return `
+    import * as path from 'path';
+    import { ExceptionFilter, Catch, NotFoundException, ArgumentsHost } from '@nestjs/common';
+
+    @Catch(NotFoundException)
```

Comment:
I'm not sure if I would recommend this solution, as I am firing the NotFoundException in my Applications when an entry can't be found in the DB too. In this case my REST API would serve me a HTML String instead of a 404 HTTP Response Code.